code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -58,15 +58,16 @@
 	if(!job) //It might be setup on spawn.
 		setup_job()
 
-	AddComponent(/datum/component/bump_attack)
+	if(!(status_flags & INCORPOREAL))
+		AddComponent(/datum/component/bump_attack)
```

Comment:
snowflake ?

Reply:
What? How is this snowflake

Reply:
literally a check if the xeno is incorporeal in xeno code for a thing only hivemind uses, add a list of components for xeno instead or something like that.